servers/physics_3d/gjk_epa.cpp

```diff
@@ -37,7 +37,7 @@
 
 /*
 Bullet Continuous Collision Detection and Physics Library
-Copyright (c) 2003-2008 Erwin Coumans  http://continuousphysics.com/Bullet/
+Copyright (c) 2003-2008 Erwin Coumans  https://bulletphysics.org
```

Comment:
We do not tamper with copyright statements.

Reply:
The old URL is completely dead. It leads to a WordPress page with this on it:

เว็บดูหนังออนไลน์ฟรี - หนัง หนังไทย ดูการ์ตูนสุดมันเพลิดเพลินใจ

Reply:
Yeah the URL is bogus, but that's not our role to update copyright statements or licenses. Those are legal aspects and we should respect what was included in the code we vendored, even if it's bogus.

For example the [CC-BY 3.0](https://creativecommons.org/licenses/by/3.0/legalcode) license has a typo in "MERCHANTIBILITY", and I've made sure to prevent `codespell` from fixing it as it's not our job to do so (and we're modifying a standard legal document if we do). Now it's been fixed in CC-BY 4.0 so 6d26ff4eda3e37f19653cf0f04d10790475cc2c1 solved it, but if we add back CC-BY 3.0 content, we'll have to respect this typo :)

> A better solution would be to ask Bullet to update the URL.

That's worth doing since they have bogus URLs in their current code. It's still debatable whether we should also update it in the 10 year old version of their code we vendor, or if we should respect whatever was their copyright statement at the time.